internal/dao/cronjob.go

```diff
@@ -98,7 +98,7 @@ func (c *CronJob) ScanSA(ctx context.Context, fqn string, wait bool) (Refs, erro
 		if err != nil {
 			return nil, errors.New("expecting CronJob resource")
 		}
-		if cj.Spec.JobTemplate.Spec.Template.Spec.ServiceAccountName == n {
+		if serviceAccountMatches(&cj.Spec.JobTemplate.Spec.Template.Spec, n) {
```

Comment:
The only thing this convenience function cares about is the serviceAcct name why not pass that in directly vs the entire spec?

Reply:
I originally did it to help with the empty string scenario but with your suggestion in the helper I think doing both strings works well. Updated.